src/components/MultiHopTrade/components/TradeConfirm/ReceiveSummary.tsx

```diff
@@ -237,41 +240,43 @@ export const ReceiveSummary: FC<ReceiveSummaryProps> = memo(
                 </Row.Value>
               </Row>
             )}
-            <>
-              <Divider borderColor='border.base' />
-              <Row>
-                <Row.Label>
-                  <Text translation={minAmountAfterSlippageTranslation} />
-                </Row.Label>
-                <Row.Value whiteSpace='nowrap'>
-                  <Stack spacing={0} alignItems='flex-end'>
-                    <Skeleton isLoaded={!isLoading}>
-                      <Amount.Crypto value={amountAfterSlippage} symbol={symbol} />
-                    </Skeleton>
-                    {isAmountPositive &&
-                      hasIntermediaryTransactionOutputs &&
-                      intermediaryTransactionOutputsParsed?.map(
-                        ({ amountCryptoPrecision, symbol, chainName }) => (
-                          <Skeleton isLoaded={!isLoading} key={`${symbol}_${chainName}`}>
-                            <Amount.Crypto
-                              value={amountCryptoPrecision}
-                              symbol={symbol}
-                              prefix={translate('trade.or')}
-                              suffix={
-                                chainName
-                                  ? translate('trade.onChainName', {
-                                      chainName,
-                                    })
-                                  : undefined
-                              }
-                            />
-                          </Skeleton>
-                        ),
-                      )}
-                  </Stack>
-                </Row.Value>
-              </Row>
-            </>
+            {swapSource !== THORCHAIN_STREAM_SWAP_SOURCE && (
```

Comment:
don't show min amount after slippage for streaming swaps as it is not applicable (we can't accurately display slippage incurred as it will be determined by the network).